tensorboard/components/tf_color_scale/test/BUILD

```diff
@@ -3,26 +3,15 @@ package(
     default_visibility = ["//tensorboard:internal"],
 )
 
-load("//tensorboard/defs:web.bzl", "tf_web_library")
-load("@io_bazel_rules_webtesting//web:py.bzl", "py_web_test_suite")
+load("//tensorboard/defs:web.bzl", "tf_web_library", "tf_web_test")
 
 licenses(["notice"])  # Apache 2.0
 
-py_web_test_suite(
+tf_web_test(
     name = "test",
-    srcs = ["test.py"],
-    browsers = ["//tensorboard/functionaltests/browsers:chromium"],
-    timeout = "short",
-    flaky = True,
-    data = [
-        ":test_web_library",
-    ],
-    srcs_version = "PY2AND3",
-    deps = [
-        "@io_bazel_rules_webtesting//testing/web",
-        "//tensorboard/functionaltests:wct_test_driver",
-    ],
-)
+    web_library = ":test_web_library",
+    src = "/tf-color-scale/test/tests.html",
+);
```

Comment:
may have added semi-colon by mistake. here and elsewhere

Reply:
Yep, thanks! Will revert.